functional_test.go

```diff
@@ -87,7 +87,7 @@ func sendRequest(url string, c *http.Cookie) error {
 func weSendARequestWithAuthorizationDataAuthorizingAccess(where, level string) error {
 	var c *http.Cookie
 	url := p.Host
-	expires := time.Now().Add(1000 + time.Second*time.Duration(rand.Intn(1000)))
+	expires := time.Now().Add(1000 + time.Second*time.Duration(rand.Intn(1000))) // #nosec G404 weak random number is OK
```

Comment:
Not looked much at it, but off the top of my head, do we even need to add a random number of seconds here?

Reply:
Not really. I suppose it's a poor man's fuzzing.

Reply:
Are you fine with removing it? I think it just adds unnecessary complexity (and complains about a possible security issue).

Reply:
I guess so. I'm sorry I didn't make that change before merging.